tests/publiccloud/check_services.pm

```diff
@@ -40,16 +40,35 @@ sub run {
     my $failed = 0;
     foreach my $service (@failed_services) {
         # 4. Capture the log output for the service into a Perl variable
-        my $log_output = $instance->ssh_script_output("sudo journalctl -u $service --no-pager");
+        my $log_output = $instance->ssh_script_output(
+            "sudo journalctl -u $service --no-pager"
+        );
 
-        # 5. Pass the captured logs as the text for record_info.
-        #    This creates a single, expandable log entry in the openQA web UI.
+        # 5. Exceptions: some services are suposed to fail, or there's an
+        # ongoing bug for them.
         if ($service =~ /systemd-vconsole-setup/) {
-            record_soft_failure("bsc#1249902 - systemd-vconsole-setup.service failed to load:\n$log_output");
-            next;
+            record_soft_failure(
+                "bsc#1249902 - systemd-vconsole-setup.service failed to load:\n$log_output"
+            );
+        }
+        elsif ($service =~ /NetworkManager-wait-online/) {
+            # Shouldn't we run some check here?
+            record_soft_failure(
+                "This service throws false positives sometimes:\n$log_output"
```

Comment:
Soft-failures are supposed to point to a poo ticket or a bsc bug.  Consider opening a ticket for this one and adjust accordingly.

Reply:
Ah ok, I thought this was a known issue... I'll create it, thanks.